src/OrchardCore/OrchardCore/Modules/ModularBackgroundService.cs

```diff
@@ -34,17 +36,24 @@ internal class ModularBackgroundService : BackgroundService
         private readonly IHttpContextAccessor _httpContextAccessor;
         private readonly ILogger _logger;
         private readonly IClock _clock;
+        private readonly bool _lazyBuildDisabled;
 
         public ModularBackgroundService(
             IShellHost shellHost,
             IHttpContextAccessor httpContextAccessor,
             ILogger<ModularBackgroundService> logger,
-            IClock clock)
+            IClock clock,
+            Microsoft.Extensions.Configuration.IConfiguration configuration)
         {
             _shellHost = shellHost;
             _httpContextAccessor = httpContextAccessor;
             _logger = logger;
             _clock = clock;
+
+            _lazyBuildDisabled = configuration
+                .GetSection("OrchardCore")
+                .GetSectionCompat("OrchardCore_Shells_LazyBuild")
+                .GetValue("Disabled", true);
```

Comment:
Shouldn’t this be false by default?

Reply:
Yes you're right, because here true means that lazy build would be `Disabled` ;)

My other little concern is that maybe it is better to make it a bg service config option

I will think about it this night